parity/cli/subcommands.rs

```diff
@@ -0,0 +1,242 @@
+use serde_derive::Deserialize;
+use structopt::StructOpt;
+
+#[derive(StructOpt, Deserialize, Debug, Clone)]
+pub enum SubCommands {
+    Daemon(Daemon),
+    Wallet {
+        #[structopt(subcommand)]
+        wallet: Wallet,
+    },
+    Account {
+        #[structopt(subcommand)]
+        account: Account,
+    },
+    Import(Import),
+    Export {
+        #[structopt(subcommand)]
+        export: Export,
+    },
+    Signer(Signer),
+    Snapshots(Snapshots),
+    Restore(Restore),
+    Tools(Tools),
+    Db(Db),
+    #[structopt(
+        about = "Print the hashed light clients headers of the given --chain (default: mainnet) in a JSON format. To be used as hardcoded headers in a genesis file."
+    )]
+    ExportHardcodedSync,
+    Dapp(Dapp),
```

Comment:
This is deprecated and not functional at all I believe, so should be safe to remove.

Reply:
Concur.

Reply:
Okay will remove this